.github/workflows/fast_testing.yml

```diff
@@ -6,6 +6,20 @@ on:
   workflow_dispatch:
 
 jobs:
+  version-check:
+    # Skip pull request job when the source branch is in the same
+    # repository. Also we need this job to run only on push with tag.
+    if: |
+      (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
```

Comment:
Well. since there is no "pull request job", this one seems excessive

Reply:
Done